### PR TITLE
add info about cache expiration

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9182#issuecomment-815945339

### CODE REVIEW

File: windows/security/threat-protection/security-policy-settings/interactive-logon-number-of-previous-logons-to-cache-in-case-domain-controller-is-not-available.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ The value of this policy setting indicates the number of users whose logon infor
 Users who access the server console will have their logon credentials cached on that server. A malicious user who is able to access the file system of the server can locate this cached information and use a brute-force attack to determine user passwords. Windows mitigates this type of attack by 
 encrypting the information and keeping the cached credentials in the system's registries, which are spread across numerous physical locations.
 
+> [!NOTE]
+> The cached account information does not expire, but can get overwritten, as described previously.
```
Comment (illfated) [inline windows/security/threat-protection/security-policy-settings/interactive-logon-number-of-previous-logons-to-cache-in-case-domain-controller-is-not-available.md:46]: Minor grammar adjustment
```suggestion
> The cached account information does not expire, but can get overwritten, as previously described.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestion.

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
@VLG17 : Would you be so kind as to commit the suggested change and make this PR ready for final review & merge?

**Review by illfated — APPROVED** [review]

**Pull request merged**